user_profile/models.py

```diff
@@ -15,4 +26,40 @@ class UserProfile(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
 
+    objects = UserProfileManager()
+
+
+@receiver(post_save, sender=User)
+def send_email_to_new_user(sender, instance, created, **kwargs):
+    if created:
+        try:
+            send_mail(
+                'New Account',
+                'Welcome to Python training Application.',
+                settings.EMAIL_HOST_USER,
+                [instance.email],
+                fail_silently=False
+            )
+        except SMTPException:
+            pass
+        print(settings.EMAIL_HOST_USER)
+        print("Email Successfully Sent to:", instance.email)
+
+
+class Settings(models.Model):
```

Comment:
`Settings` is not a good name for model. Invitation for conflicts.

Reply:
Changed.